frontend/src/components/CreateGift/Erc20Select.tsx

```diff
@@ -51,6 +51,15 @@ export const erc20TokensData = [
     name: "y3Crv",
     symbol: "y3Crv",
   },
+  {
+    address: "0x6b3595068778dd592e39a122f4f5a5cf09c90fe2",
+    chainId: 1,
+    decimals: 18,
+    logoURI:
+      "https://raw.githubusercontent.com/sushiswapclassic/token-list/master/assets/SUSHI.png",
```

Comment:
@milkyklim do we want to pull images from here instead?

https://github.com/iearn-finance/yearn-assets/blob/master/icons/tokens/0x6B3595068778DD592e39A122f4f5a5cF09C90fE2/logo-128.png?raw=true